app/assets/src/components/views/SampleUploadFlow/LocalSampleFileUpload.jsx

```diff
@@ -129,6 +129,12 @@ class LocalSampleFileUpload extends React.Component {
                 Paired files must be labeled with &quot;_R1&quot; or
                 &quot;_R2&quot; at the end of the basename.
               </li>
+              <li>
+                File names must be no longer than 120 characters and can only
```

Comment:
Can you make this text a constant for reuse?

Reply:
You mean, the entire text or just this new sentence? Because it seems there are other lines that can be reused as well.
Do you think this is a blocker for this PR or is it an sanitization that we should reconsider in a PR dedicated to remove such redundancies?

Reply:
I would prefer to do it since it is a new sentence. It is not the problem of redundancy but keeping it in sync. 
I see your point but I prefer to start doing it and set an example than keep following a wrong pattern.

Reply:
Considering this is not only about this sentence, but the entire block of instructions (the previous 2 `<li>` items are exactly the same), would it make sense to create a new react component with these instructions instead of using a constant?

Reply:
Do we have any pre-existing pattern that I should follow in this case?

Reply:
Ok, you got me convinced. Let's just make a low priority jira task to just consolidate this

Reply:
Ok, you got me convinced. Let's just make a low priority jira task to just consolidate this

Reply:
Ok, you got me convinced. Let's just make a low priority jira task to just consolidate this